Assets/PatchKit Patcher/Editor/CustomBuilding.cs

```diff
@@ -78,6 +78,25 @@ private static string PatcherExecutableName(BuildTarget target)
             }
         }
 
+        private static void EnforceCorrectScreenSizing()
+        {
+            if (PlayerSettings.defaultScreenWidth != Constants.ScreenWidth
```

Comment:
I think this is a breaking change as we recommend to change player settings to adjust the patcher resolution.